Dockerfile

```diff
@@ -5,9 +5,11 @@ RUN apk add --no-cache bash
 # Setting working directory.
 WORKDIR /usr/src/app
 
+RUN corepack enable yarn
+
 # Installing dependencies
-COPY package*.json yarn.lock ./
-RUN yarn install
+COPY package*.json yarn.lock .yarnrc.yml ./
```

Comment:
`.yarnrc.yml` is required to ensure that yarn installs into `node_modules` folder